CONTRIBUTING.md

```diff
@@ -86,10 +92,6 @@ Please keep your PRs manageable and easy to review.
 
 ## Suggested debugging workflow
 
-For development purposes, it is possible to manually perform an incremental build simply by bypassing Maven and Gradle entirely and
```

Comment:
I see that the documentation for using `uf.py` is removed, but the script itself isn't. Why remove the docs if the script is still there? If forget what it's for, but @petervdonovan or @a-sr would know.

Reply:
The docs are still present on line 110; they are just less prominent and recommend calling it through NPM. The reason for the docs is to do this incremental build that essentially manually patches the JAR. It's a hacky workaround for long compile times.

Reply:
Got it -- I missed that it was still being referenced by the `amend-jar` script.